packages/web/app/tailwind.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'tailwindcss/lib/util/flattenColorPalette' {
```

Comment:
We already have this

```ts
declare module 'tailwindcss/lib/util/flattenColorPalette' {
  export default function flattenColorPalette(
    pallette: Record<string, string>,
  ): Record<string, string>;
}
```

in `packages/web/docs/src/deps.d.ts`

Maybe we could have one file with Tailwind type module augmentation included in both projects in `web`?